### PR TITLE
Fixed code warnings

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

File: execute_wsh_site.h [hunk 1]
```diff
@@ -91,7 +91,7 @@ static HRESULT STDMETHODCALLTYPE active_script_site_on_script_error(IActiveScrip
     // Print exception information to log
     HRESULT result = IActiveScriptError_GetExceptionInfo(error, &excep_info);
     if (FAILED(result)) {
-        LOG_ERROR("Failed to get active script error (0x%08x)\n", result);
+        LOG_ERROR("Failed to get active script error (0x%08lx)\n", result);
```
Comment (sergio-nsk) [inline execute_wsh_site.h:94]: Why don't use `PRI*` format dpecifiers?

File: test/test_wpad.cc [hunk 2]
```diff
@@ -39,7 +39,8 @@ TEST(wpad, dhcp) {
     EXPECT_NE(localent, nullptr);
 
     // Create network adapter
-    net_adapter_s adapter = {0};
+    net_adapter_s adapter;
+    memset(&adapter, 0, sizeof(adapter));
```
Comment (sergio-nsk) [inline test/test_wpad.cc:43]: What is inner struct in that type? I suppose it was a warning about aggregate initialization.
It should be
```c++
net_adapter_s adapter{};
```

**Review by sergio-nsk — CHANGES_REQUESTED** [review]

File: test/test_wpad.cc (hunk 2 above)
Comment (nmoinvaz) [reply test/test_wpad.cc]: Fixed.

File: execute_wsh_site.h (hunk 1 above)
Comment (nmoinvaz) [reply execute_wsh_site.h:94]: They don't have `lx` only `llx`

**Review by sergio-nsk — APPROVED** [review]

**Pull request merged**